cfme/tests/test_rest_manual.py

```diff
@@ -267,7 +275,9 @@ def test_delete_vm_disk_via_api():
         expectedResults:
             1. The disk must be deleted successfully.
 
-    Bugzilla: 1666593, 1620161
+    Bugzilla:
+        1666593
+        1620161
```

Comment:
Question: 
what is wrong in keeping it like this: ```Bugzilla: 1666593, 1620161```

Reply:
trying to keep it consistent, make changes 1-line, and avoid weird formatting on longer lists of BZ's.

I would actually like to script moving all of these to `pytest.mark.meta(automates=[BZ()]` decorators.